checks/rabbitmq/check.go

```diff
@@ -119,6 +125,10 @@ func New(config Config) func(ctx context.Context) error {
 			case <-time.After(config.ConsumeTimeout):
 				checkErr = fmt.Errorf("RabbitMQ health check failed due to consume timeout: %w", err)
 				return
+			case <-ctx.Done():
+				checkErr = fmt.Errorf("RabbitMQ health check failed due "+
+					"to health check listener disconnect: %s", ctx.Err())
```

Comment:
It makes sense to wrap original error and let clients to decide if they need it or now

```suggestion
				checkErr = fmt.Errorf("RabbitMQ health check failed due "+
					"to listener disconnect: %w", ctx.Err())
```